### PR TITLE
Fixes thralls having no examine message

### PR DESCRIPTION
**Pull request by ghost**
How long has this been removed?

:cl: steamp0rt
fix: Fixes thralls having no examine message
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/examine.dm [hunk 1]
```diff
@@ -88,6 +88,8 @@
 
 		if(digitalcamo)
 			msg += "[t_He] [t_is] moving [t_his] body in an unnatural and blatantly unsimian manner.\n"
+	
+	msg += handleThrallMsg(src) // HIPPIE -- add thrall examine message
```
Comment (Carbonhell) [inline code/modules/mob/living/carbon/examine.dm]: what is this?

Comment (ghost) [reply code/modules/mob/living/carbon/examine.dm]: it's a hook that handles the examine text: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/code/modules/mob/living/carbon/examine.dm#L1-L5

**Comment by ghost** [comment]
oh nice, travis failed due to ~~stalled unit tests~~ solar flares

**Comment by Carbonhell** [comment]
i do not like this at all
will check if there's a better way to do this

**Review by Carbonhell — CHANGES_REQUESTED** [review]
no

**Comment by ghost** [comment]
This is how it originally did it, though?

This is pretty much how we handle all text-based things, hooks?

File: code/modules/mob/living/carbon/examine.dm [hunk 2]
```diff
@@ -88,6 +88,8 @@
 
 		if(digitalcamo)
 			msg += "[t_He] [t_is] moving [t_his] body in an unnatural and blatantly unsimian manner.\n"
+	
+	msg += hippie_examine(src) // hippie -- add our specific examine stuff
```
Comment (Carbonhell) [inline code/modules/mob/living/carbon/examine.dm]: what is the point of passing src? You already have access to it with....src??

File: hippiestation/code/modules/mob/living/carbon/examine.dm [hunk 3]
```diff
@@ -1,5 +1,5 @@
-/mob/living/carbon/proc/handleThrallMsg(sr)
+/mob/living/carbon/proc/hippie_examine(sr)
```
Comment (Carbonhell) [inline hippiestation/code/modules/mob/living/carbon/examine.dm]: Perhaps rename to hippie_carbon_examine

File: hippiestation/code/modules/mob/living/carbon/examine.dm [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
 	var/mob/living/carbon/A = sr
 	if(!A.wear_mask && is_thrall(A))
```
Comment (Carbonhell) [inline hippiestation/code/modules/mob/living/carbon/examine.dm]: Just use src??

**Review by Carbonhell — DISMISSED** [review]

File: code/modules/mob/living/carbon/examine.dm (hunk 2 above)
Comment (ghost) [reply code/modules/mob/living/carbon/examine.dm]: Hell if I know, this is oldcode. Will fix

**Comment by ghost** [comment]
Done.

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**